changelog.d/20231212_095641_yadudoc1729_update_to_parsl_2023_12_4.rst

```diff
@@ -0,0 +1,35 @@
+.. A new scriv changelog fragment.
+..
+.. Uncomment the header that is right (remove the leading dots).
+..
+.. New Functionality
+.. ^^^^^^^^^^^^^^^^^
+..
+.. - A bullet item for the New Functionality category.
+..
+.. Bug Fixes
+.. ^^^^^^^^^
+..
+.. - A bullet item for the Bug Fixes category.
+..
+.. Removed
+.. ^^^^^^^
+..
+.. - A bullet item for the Removed category.
+..
+.. Deprecated
+.. ^^^^^^^^^^
+..
+.. - A bullet item for the Deprecated category.
+..
+Changed
+^^^^^^^
+
+- Parsl version requirements updated from 2023.7.3 to 2023.12.4
+
```

Comment:
Should we add some details on what was added/fixed in Parsl?

Reply:
I'm not worried about that from the Globus Compute side.  That's a detail for the Parsl project, no?

What are you thinking?

Reply:
Yeah, I suppose that's the right tack. I was thinking we might mention some key highlights that directly affect Globus Compute (e.g., improvements to worker management).